ci-operator/config/openshift/secrets-store-csi-driver-operator/openshift-secrets-store-csi-driver-operator-main.yaml

```diff
@@ -107,6 +107,19 @@ tests:
       OO_PACKAGE: secrets-store-csi-driver-operator
       OO_TARGET_NAMESPACES: '!all'
     workflow: openshift-e2e-azure-csi-secrets-store
+- as: operator-e2e-vault
+  optional: true
+  steps:
+    cluster_profile: azure4
+    dependencies:
+      OO_INDEX: ci-index-secrets-store-csi-driver-operator-bundle
+      SECRETS_STORE_E2E_PROVIDER_IMAGE: secrets-store-csi-driver-e2eprovider
+    env:
+      OO_CHANNEL: preview
+      OO_INSTALL_NAMESPACE: openshift-cluster-csi-drivers
+      OO_PACKAGE: secrets-store-csi-driver-operator
+      OO_TARGET_NAMESPACES: '!all'
+    workflow: openshift-e2e-azure-csi-secrets-store-vault
```

Comment:
Use the workflow, we need add the `secrets-store-csi-driver-testvault` image to base_images.
E.g.
https://github.com/openshift/release/pull/50084/files#diff-2c0a7b64192c771610e79d45c1ff783e4bae5ba5db161d0fc49de984920836afR30-R33